src/util/composables.ts

```diff
@@ -55,29 +63,17 @@ export const useQuery = <Data, Args extends Array<unknown>>(
 
         /** on success callback */
         if (onSuccess) onSuccess(result, args);
-
-        /** return results */
-        return result;
       } else {
-        /** otherwise, throw special "stale" error */
-        throw new Error("Stale query");
+        /** otherwise, log special "stale" error */
+        console.error("Stale query");
```

Comment:
Again, just checking, but I assume you switched from throwing an exception to logging since de-staling is no longer something you're including in your useQuery implementation, right?

Reply:
This change should only be a refactor, it shouldn't change the behavior at all. I noticed that I was throwing a "stale" error in the try block, detecting it in the catch block by matching the string "stale", then just logging the error. Now, instead, I just log the error in place to clean up the code.

Reply:
Aha, that makes sense. Thanks!